lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,141 @@
+---
+name: Organization
+field_name_base_url: https://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: The identifier for the Organization.
+    children:
+
+      - name: type
+        required: 'No'
+        cardinality: 0..1
+        type: CodeableConcept
+        description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+        url: https://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+        binding:
+          description: The type used to determine which identifier to use for a specific purpose.
+          terminology:
+            - display: Identifier Type Codes
+              system: http://hl7.org/fhir/identifier-type
+              info_link: http://terminology.hl7.org/CodeSystem/v2-0203.html
+            - display: v2 Identifier Type
+              system: http://terminology.hl7.org/CodeSystem/v2-0203
+              info_link: https://hl7.org/fhir/R4/v2/0203/index.html
+            - display: Millennium Organization Identifier (Alias) Types
+              system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/334
+              info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-334-organization-alias-type
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: CodeableConcept
+    description: The type for the Organization.
+    binding:
+      description: Indicates the type of Organization.
+      terminology:
+        - display: OrganizationType
+          system: http://hl7.org/fhir/ValueSet/organization-type
+          info_link: http://terminology.hl7.org/CodeSystem/organization-type.html
+        - display: Millennium Organization Identifier (Alias) Types
+          system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/278
+          info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-278-organization-type
+
+  - name: resourceType
+    required: 'Yes'
+    cardinality: 1..1
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Organization"
+      }
+    note: resourceType must be Organization.
+    url: https://hl7.org/fhir/R4/json.html#resources
+
+  - name: identifier
```

Comment:
So these should mix into the read ones. Shouldnt need 2

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R9